clients/slppclient/slpp_client.go

```diff
@@ -165,3 +167,9 @@ func (c SlppClient) GetActions() (models.Actions, error) {
 func getSlppURL(org, space, serviceID, processID string) string {
 	return "slprot/" + org + "/" + space + "/slp/runs/" + serviceID + "/" + processID
 }
+
+func getSlppURLEncoded(org, space, serviceID, processID string) string {
+	orgEncoded := strings.Replace(url.QueryEscape(org), "+", "%20", -1)
```

Comment:
Same here?

Reply:
Only original function getRestURL is kept